incubator/binding-grpc/pom.xml

```diff
@@ -26,7 +26,7 @@
   <properties>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
-    <jacoco.coverage.ratio>0.90</jacoco.coverage.ratio>
+    <jacoco.coverage.ratio>0.58</jacoco.coverage.ratio>
```

Comment:
This will need to be adjusted later when code coverage improves after adding ITs.